CLIENTS.md

```diff
@@ -0,0 +1,82 @@
+# Featured Client Requirements
+Authored by: ShowierData9978 <contact@showierdata.xyz> on 4/18/24 (CT).
+
+## Terms/Definitions
+
+- Us/We - The Meower team
+- OSI - Open Source Initiative
+
+## Rules
+- Clients created for malicious intent will not be featured
+- Clients must be Open Source Software (Any OSI approved licence).
+- If your client implements any kind of plugin/addon interface, any plugins that are featured (or in the repo) are considered to be part of the client.
+- The Meower Team may, at it's sole discretion, choose whether or not a client should be featured
+- The Meower Team may feature a client despite not complying with these guidelines.
+
+## Guidelines
+
+### Design Practices
+
+- Cache chat and post data
```

Comment:
very vague, this basically allows clients like meower95 to store deleted messages, a feature multiple times said as "not complying with RFC".